### PR TITLE
[TD-10610]<fix>: push sms alert.

### PR DESCRIPTION
**Pull request by afwerar**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -30,6 +30,12 @@ module.exports = function (grunt) {
         expand: true,
         src: ['README.md'],
         dest: 'dist'
+      },
+      cmsConfig: {
```
Comment (sangshuduo) [inline Gruntfile.js:34]: smsConfig?

File: Gruntfile.js [hunk 2]
```diff
@@ -87,5 +93,5 @@ module.exports = function (grunt) {
   });
 
   grunt.registerTask('default', ['clean', 'copy:src_to_dist', 'copy:dashboard_to_dist',
-    'copy:examples_to_dist', 'copy:pluginDef', 'babel', 'mochaTest']);
+    'copy:examples_to_dist', 'copy:pluginDef', 'copy:cmsConfig', 'babel', 'mochaTest']);
```
Comment (sangshuduo) [inline Gruntfile.js:96]: smsConfig?

Comment (afwerar) [reply Gruntfile.js:96]: The plug-in needs to configure the secret key of the SMS platform and push the mobile phone number, which are stored in the external json file.

Comment (zitsen) [reply Gruntfile.js:96]: typo: cms -> sms

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**